observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -24,6 +24,45 @@ The following table provides information about which features are available depe
 | 1.1+          | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
 |===
 
+[id="cluster-observability-operator-release-notes-1-2_{context}"]
+== {coo-full} 1.2
+
+// Back fill advisory when it is created by KOnflux
+// The following advisory is available for {coo-full} 1.1.1:
+//
+// * link:https://access.redhat.com/errata/RHSA-2025:????[RHEA-2025:??? {coo-full} 1.1.1]
```

Comment:
This should be 1.2.0

Reply:
I changed the comment in the source but this won't trigger a rebuild